app/views/external_users/claims/fixed_fees/advocates/_fixed_fee_fields.html.haml

```diff
@@ -18,14 +18,14 @@
     - if f.object.fee_type.case_uplift?
       .form-group
         = f.adp_text_field :case_numbers,
-          label: t('.case_numbers'),
+          label: t('.case_numbers_html', context: f.object.description),
           input_classes:'js-fixed-fee-case-numbers',
           hint_text:'Separate by commas',
           errors: @error_presenter
 
     .form-group
       = f.adp_text_field :quantity,
-        label: t('.quantity'),
+        label: t('.quantity_html', context: f.object.description),
```

Comment:
not a blocker, and i like the use of `context`, but since it is not using a translation it will always be english - not that we are or probably ever will add another language (welsh 😬 )